package.json

```diff
@@ -268,7 +268,7 @@
     "@department-of-veterans-affairs/va-forms-system-core": "1.6.1",
     "@department-of-veterans-affairs/vagov-platform": "^0.0.1",
     "@mapbox/mapbox-sdk": "~0.13.2",
-    "@reduxjs/toolkit": "^2.2.5",
+    "@reduxjs/toolkit": "^1.9.5",
```

Comment:
Had to downgrade to allow the unit tests to pass. It was having an issue with Node 14. Here is the Slack conversation regarding it being okay to downgrade redux toolkit.

https://dsva.slack.com/archives/CBU0KDSB1/p1740159417985599